src/types.ts

```diff
@@ -11,6 +11,7 @@ export type ValueDefinitionValue = string | number | bigint | boolean | object |
 export type ValueDefinition = {
   type: string
   default: ValueDefinitionValue
+  loc?: any
```

Comment:
Let's use a proper type for this, something like:
```suggestion
  loc?: Acorn.SourceLocation | null,
```